Introduced first version of Release Info Tool

HolQue opened the pull request:
Currently support of Windows only, because of:
https://github.com/test-fullautomation/RobotFramework_AIO/issues/165

Requires
https://github.com/test-fullautomation/python-extensions-collection/pull/43

HolQue committed:
Introduced first version of Release Info Tool

Currently support of Windows only, because of:
https://github.com/test-fullautomation/RobotFramework_AIO/issues/165
Requires
https://github.com/test-fullautomation/python-extensions-collection/pull/43

HolQue committed:
Merge branch 'develop' into HolQue/task/release_info_tool

test-fullautomation reviewed (changes requested):
Hi Holger,

1. you wanted to add (link with) the files of the repositories. here we have still the release_items* as part of the pull-request.

2. robotframework core, robotframework documentation, ... (please check all repos) also need a release_item file.

Please provide a corresponding pull-request.

Thank you,
 Thomas

HolQue committed:
Added further release_items files to configuration file release_info_config_OSS.json

Introduced usage of environment variable 'RobotPythonSitePackagesPath' in configuration file
This requires the pull requests in all other repositories where those release_items files are added

test-fullautomation reviewed (approved):
Hi Holger,
looks good to me!
Thank you,
 Thomas
